checkov/kubernetes/checks/resource/base_spec_check.py

```diff
@@ -50,6 +50,6 @@ def wrapper(self, conf, entity_type=None):
     @staticmethod
     def get_inner_entry(conf: Dict[str, Any], entry_name: str) -> Dict[str, Any]:
         spec = {}
-        if conf.get("spec") and conf.get("spec").get("template"):
+        if conf.get("spec") and isinstance(conf["spec"], dict) and conf.get("spec").get("template"):
```

Comment:
```suggestion
        if conf.get("spec") and conf.get("spec", {}).get("template"):
```

Reply:
this doesn't solve the issue - it returns a None object and does not use the default empty dict